source/collimation.f90

```diff
@@ -1834,6 +1837,7 @@ subroutine collimate_start
           "7=x_in(b!)[m], 8=xp_in, 9=y_in, 10=yp_in, 11=x_out [m], 12=xp_out, 13=y_out, 14=yp_out"
         write(coll_scatter_unit,"(a)") "# 1=icoll, 2=iturn, 3=np, 4=nabs (1:Nuclear-Inelastic,2:Nuclear-Elastic,3:pp-Elastic, "//&
           "4:Single-Diffractive,5:Coulomb), 5=dp, 6=dx', 7=dy'"
+        write(profiling_jaw_unit,"(a)") "# 1=icoll, 2=iturn, 3=np, 4=x, 5=xp, 6=y, 7=yp, 8=s, 9=[1:in,2:out,3:back]"
```

Comment:
Can we please have the column headers aligned instead of numbered? We've been updating a lot of files to do this, so I would prefer it if we didn't add new files that needs later updating.